plugins/libvirt/kvm_io

```diff
@@ -59,54 +63,59 @@ def clean_vm_name(vm_name):
             pass
     return re.sub(r"[^a-zA-Z0-9_]", "_", vm_name)
 
+
 def fetch(vms):
     ''' Fetch values for a list of pids
     @param dictionary {kvm_pid: cleaned vm name}
     '''
     res = {}
     for pid in vms:
-        f = open("/proc/%s/io" % pid, "r")
+        f = open("/proc/%s/io" % pid.decode("utf-8"), "r")
```

Comment:
You may shorten this to `.decode()` - but this is just a matter of style.
In addition you may like to switch to `with open(...) as f:`?

(feel free to ignore these suggestions)